crates/rspack_core/src/options/module.rs

```diff
@@ -193,6 +193,22 @@ impl ExportPresenceMode {
   }
 }
 
+#[derive(Debug, Clone, Copy, MergeFrom)]
+pub enum OverrideStrict {
+  Strict,
+  NoneStrict,
+}
+
+impl From<&str> for OverrideStrict {
+  fn from(value: &str) -> Self {
+    match value {
+      "strict" => Self::Strict,
+      "non-strict" => Self::NoneStrict,
+      _ => unreachable!("parser.overrideStrict should be 'strict' or 'non-strict'"),
```

Comment:
I'm not sure if it is a good choice to use `unreachable!`

Reply:
It's ok, invalid configuration will be rejected by zod